docs/install.md

```diff
@@ -27,37 +27,44 @@ Yazi can be _optionally_ extended with other command line tools to enable additi
 sudo pacman -S yazi ffmpegthumbnailer unarchiver jq poppler fd ripgrep fzf zoxide
 ```
 
-If you want to use the latest git version, you can install it from [AUR](https://aur.archlinux.org/packages/yazi-git/) or [Arch Linux CN](https://github.com/archlinuxcn/repo/):
+If you want to use the latest Git version, you can install it from [AUR](https://aur.archlinux.org/packages/yazi-git/) or [Arch Linux CN](https://github.com/archlinuxcn/repo/):
 
 ```bash
 paru -S yazi-git ffmpegthumbnailer unarchiver jq poppler fd ripgrep fzf zoxide
 ```
 
-## macOS
+## macOS / Homebrew
 
-Install Yazi and its dependencies with Homebrew:
+First, make sure that Homebrew is fully up-to-date with `brew update`.
+
+Then you can install Yazi by itself:
+
+```bash
+brew install yazi
+```
+
+Or along with the optional dependencies:
 
 ```bash
-brew update
 brew install yazi ffmpegthumbnailer unar jq poppler fd ripgrep fzf zoxide
 brew tap homebrew/cask-fonts && brew install --cask font-symbols-only-nerd-font
 ```
 
-If you prefer to use the most recent code, use `--HEAD` flag:
+If you prefer to use the most recent code, use the `--HEAD` flag when installing Yazi.
 
 ```bash
 brew install yazi --HEAD
 ```
 
 ## Nix
 
-The [Nix package of Yazi](https://search.nixos.org/packages?channel=unstable&show=yazi) is available. Nix users can install Yazi via:
+A [Nix package of Yazi](https://search.nixos.org/packages?channel=unstable&show=yazi) is available. Nix users can install Yazi via:
```

Comment:
this is the official Nix package for Yazi, limited to just one

Reply:
Yeah but "the" doesn't work grammatically here.